docs/source/leapp-tool.rst

```diff
@@ -53,23 +53,25 @@ migrate-machine
 destroy-containers
```

Comment:
Will do, the list-machines command removal is actually still hanging on something (PR 147) and do not include docs either. I was actually thinking of removing that in the very same PR, but I would rather keep it separated. And about the check-target, I am planning to add it also in different PR. I correct the typos and continue with merging the changes.